airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/debezium/internals/SnapshotMetadata.java

```diff
@@ -16,7 +16,8 @@ public enum SnapshotMetadata {
   LAST_IN_DATA_COLLECTION,
   TRUE,
   LAST,
-  FALSE;
+  FALSE,
+  NULL;
```

Comment:
Just checking: should `NULL` be in `ENTRIES_OF_SNAPSHOT_EVENTS`? I don't actually know.

Reply:
Because the change in json converter affects also the conversion of the metadata potion of a ChangeEvent, the value of the `"snapshot": …` key turned from false to null.